webhook-deprecations.rst

```diff
@@ -0,0 +1,41 @@
+.. post:: Jan 11, 2019
+    :tags: deprecation
+    :author: Anthony
+
+Incoming Webhook Deprecations
+=============================
+
+In the coming weeks and months, Read the Docs will be moving some projects away
+from our legacy incoming webhooks. About 2 years ago, these incoming webhooks
+were replaced with project webhook integrations, at which point we improved
+security and added several features to these incoming webhooks. New projects
+have been using these integrations by default since, but it's possible that
+your project is still using these deprecated incoming webhooks if your project
+was configured on Read the Docs more than 2 years ago.
+
+In order to continue building automatically, projects should reconfigure their
+repository to use a new project webhook integration instead. For more
+information, see our docs on :ref:`readthedocs:webhook-creation`.
```

Comment:
I was going to say that we should probably just have the basics of the docs inline here but the process is somewhat complicated. Are you worried about a significant dropoff in projects?

Reply:
I think we should be able to make sure most projects are configured. We'll no more on numbers once we wait a week and see who has an integration.